crates/rome_rowan/src/green/node_cache.rs

```diff
@@ -80,6 +80,16 @@ fn element_id(elem: GreenElementRef<'_>) -> *const () {
     }
 }
 
+/// The [LiveSet] tracks which nodes and tokens are reachable in the most
+/// recent revision of the syntax tree. It is filled as the syntax tree gets
+/// rebuilt, and is used to evit unused entries from the cache at the end of a
+/// parsing session.
```

Comment:
```suggestion
/// The [LiveSet] tracks which nodes and tokens are reachable in the most
/// recent revision of the syntax tree. It is filled as the syntax tree gets
/// rebuilt and is used to evict unused entries from the cache at the end of a
/// parsing session.
```

I find the term "reachable" a bit vague/unclear. What does "reachable" mean in this context?
Also, would you mind explaining a bit more why **pointers** are stored in this data structure?